pkg/apis/scheduling/types.go

```diff
@@ -353,6 +353,10 @@ type QueueSpec struct {
 	// The amount of resources configured by the user. This part of resource can be shared with other queues and reclaimed back.
 	// +optional
 	Deserved v1.ResourceList `json:"deserved,omitempty" protobuf:"bytes,9,opt,name=deserved"`
+
+	// Priority define the importance of queue
```

Comment:
Change importance to priority is better, and we can add more description like higer value get a higher priority when scheduling and preemption: )

Reply:
DONE